### DISCUSSION TITLE
Does config-watched also watches enrichment table for file changes?

### DISCUSSION THREAD

**Discussion by nmiculinic** [vectordotdev/vector#21847] [Q&A] [answered]
I'm using https://vector.dev/highlights/2021-11-18-csv-enrichment/ and am running vector with `--watch-config` flag. 

If underlying CSV file changes, will the config be reloaded or not?

**Comment by pront** [comment, marked as answer]
Hello, not at the moment. There is an open feature request for this: https://github.com/vectordotdev/vector/issues/11817.
